packages/paste-core/components/screen-reader-only/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from '@emotion/styled';
 
-export interface ScreenReaderOnlyProps {
+export interface ScreenReaderOnlyProps extends React.InputHTMLAttributes<HTMLInputElement> {
```

Comment:
Not sure how I feel about this. Can you try something similar to what we do in `Box`? We recast the styled div as a Functional React component to blow away the div typings. https://github.com/twilio-labs/paste/blob/master/packages/paste-core/primitives/box/src/index.tsx#L166

Reply:
Reverted this change.